Mathlib/Computability/TuringMachine.lean

```diff
@@ -343,11 +340,9 @@ instance Stmt.inhabited [Inhabited Γ] : Inhabited Stmt₀ :=
   the initial state. -/
 @[nolint unusedArguments] -- this is a deliberate addition, see comment
```

Comment:
Maybe off-topic, but surely we can just remove the `Inhabited Λ` argument here, and keep it everywhere else?

Reply:
Yes, sounds worth pursuing in a separate PR.